### PR TITLE
New docs

### PR DESCRIPTION
**Pull request by ralph-pichler**
New `README.md`. Should only be merged after #79 and #78

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,151 +2,78 @@
...
+`SimpleSwap` is a chequebook-style contract with support hard deposits.
 
-#### OracleTrial
+#### Cheques
 
-In the `OracleTrial` there a two `OracleWitness`es. An `OracleWitness` is a `Witness` meant for testing whose testimony for some `noteId` can be set in advance. For the `OracleTrial` to reach a `GUILTY` verdict both of the witnesses need to return a `VALID` testimony. In all other cases the trial ends with `NOT_GUILTY`.
+The owner can issue cheques. Those are a signed piece of data containing the swap address, a beneficiary, and a cumulative amount. If the beneficiary wishes to cash the cheque it needs to be sent to the contract using the `cashChequeBeneficiary` function. Alternatively anyone else can also cash the cheque using `cashCheque` provided they have a signature from the beneficiary. In that case they might also get a portion of the payout as a fee. If there is not enough liquid balance in the contract part of the cheque might bounce (whether the full cheque was paid out should be verified through the `ChequeCashed` event). Later the beneficiary can try cashing again to get the remaining amount.
```
Comment (Eknir) [inline README.md]: Owner is not a used variable anymore. I would rephrase to correctly introduce `issuer`

File: README.md [hunk 2]
```diff
@@ -2,151 +2,78 @@
...
 
-#### SimpleTrial with HashWitness
+#### Hard Deposit
 
-A `SimpleTrial` only has a single stage. The `witness` is passed in as a constructor argument.
+The owner can lock a certain amount of the balance to a specific beneficiary (`increaseHardDeposit`) to give some solvency guarantees. Decreasing a hard deposit is a two step process: First it needs to be prepared (`prepareDecreaseHardDeposit`) to start a security delay, which the beneficiary can use to cash the outstanding cheques. Afterwards the deposit can be decreased (`decreaseHardDeposit`).
```
Comment (Eknir) [inline README.md]: > to give some solvency guarantees

I propose to change this to:
`to give solvency guarantees up to the amount of the hard-deposit`

File: README.md (hunk 1 above)
Comment (ralph-pichler) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (ralph-pichler) [reply README.md]: fixed
